### ISSUE TITLE
what is the val acc when pre-training on the Miniimagenet dataset is finished?

### ISSUE THREAD

**Issue by itiandong** [Tsingularity/FRN#1]
Thank you for your great work!
Could you please tell me that on the Miniimagenet dataset, what is the highest val accuracy during pre-training, and at what epoch did it achieve it?

**Comment by Tsingularity** [comment]
Hi, thanks for your interest in our work!

For pre-training on mini-ImageNet, on my end, it gets the highest validation accuracy (62.08 ± 0.20) at the end of the whole pre-training process, which is the 350 epoch.

Feel free to let me know if there's anything else I can help with.
